### DISCUSSION TITLE
Behavior editing, patches, plugins, graph injection, with Pandora

### DISCUSSION THREAD

**Discussion by OpheliaComplex** [Monitor221hz/Pandora-Behaviour-Engine-Plus#361] [Q&A]
Would be nice with a discussion thread for creating/authoring patches, plugins, graph injection with Pandora.

Are there any example mods around showing how to inject behavior Pandora style, and not by using FNIS-style Lists, as described in the readme:

`To inject a custom graph reference, including its variables and animations into a specific graph, make a subfolder named inject in the identifying folder of the behavior graph that you want to inject into. Then, in the inject folder, make another folder that has the same name as the hkbStateMachine that you want to inject under. Then place the custom graph under this folder.`

I want to inject a statemachine, clip generator, transitions and cliptriggerarray into MT_behavior, similar to a furniture animation (fu), however those templates are not quite what I want, I'd like to tweak it some. I was hoping graph injection is how I would solve this.

Are there any examples around of these approaches?

**Comment by NachtSterben** [comment]
Hello. If I don't misunderstanding, you maked a patch to a Crouch Sliding mod for Pandora ([a fix](https://www.nexusmods.com/skyrimspecialedition/mods/112320/?tab=description)), but I still Sliding at one place. Maybe patch is wrong, maybe I don't understand how to use patch with Pandora.
